internal/catalogmetadata/types_test.go

```diff
@@ -123,7 +123,13 @@ func TestBundleRequiredPackages(t *testing.T) {
 	} {
 		t.Run(tt.name, func(t *testing.T) {
 			packages, err := tt.bundle.RequiredPackages()
-			assert.Equal(t, tt.wantRequiredPackages, packages)
+			assert.Equal(t, len(tt.wantRequiredPackages), len(packages))
+			for i, pkg := range packages {
+				// Must custom compare due to Semver function type
+				assert.Equal(t, tt.wantRequiredPackages[i].PackageRequired, pkg.PackageRequired)
```

Comment:
Haven't looked at the full test ... are these guaranteed to sort consistently (packages and tt.wantRequiredPackages)?

Reply:
It is guaranteed however I think this fix should be a separate PR. Created #423 for this.